lib/lib.es6.d.ts

```diff
@@ -8530,6 +8530,7 @@ interface DOMException {
     readonly URL_MISMATCH_ERR: number;
     readonly VALIDATION_ERR: number;
     readonly WRONG_DOCUMENT_ERR: number;
+    new(message: string, type: DOMExceptionType): DOMException;
```

Comment:
Should the second parameter just be string since in the document, it says that "Error Names" listed are not complete but just common one (https://developer.mozilla.org/en-US/docs/Web/API/DOMException)